src/dmd/dsymbolsem.d

```diff
@@ -1786,6 +1786,10 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             for (size_t i = 0; i < scd.decl.dim; i++)
             {
                 Dsymbol s = (*scd.decl)[i];
+                // https://issues.dlang.org/show_bug.cgi?id=19122
+                if (sc.inunion)
+                    if (auto decl = s.isVarDeclaration())
```

Comment:
Stylistic nit, give the first if braces?

Reply:
This only superficially works, because decls can be not symbols but can contain symbols, such as:
```
 union { align(1) { int a; long b; } }
```